trompace/queries/__init__.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+
+class StringConstant:
```

Comment:
there's no reason to copy this file - we should have these definitions in only one place in the library

Reply:
Done for this branch, but will have to do this across branches, otherwise we'll get a lot of conflicts.